### PR TITLE
feat(docs/basic/constructions.md): 增加例题 4 的题目链接

### PR DESCRIPTION
**Pull request by Estrella-Explore**
- [x] 我已认真阅读贡献指南 (contributing guidelines) 和社区公约 (code of conduct)，并遵循了如何参与页及格式手册页的相应规范。

为例题 4 加入了[题目链接](https://vjudge.net/problem/%E9%BB%91%E6%9A%97%E7%88%86%E7%82%B8-4971)

### CODE REVIEW

**Comment by toprise** [comment]
把标题改成 `feat(docs/basic/constructions.md): 增加例题 4 的题目链接` 应该可以通过 `Lint PR Title / Validate PR title (pull_request)` 检查

**Comment by Estrella-Explore** [comment]
> 把标题改成 `feat(docs/basic/constructions.md): 增加例题 4 的题目链接` 应该可以通过 `Lint PR Title / Validate PR title (pull_request)` 检查

@toprise thx

**Comment by toprise** [comment]
这个是 CI 的问题

> > 把标题改成 `feat(docs/basic/constructions.md): 增加例题 4 的题目链接` 应该可以通过 `Lint PR Title / Validate PR title (pull_request)` 检查
> 
> @toprise thx
> 
> 不过修改 pr 标题之后怎么重新跑一遍 check 呀
> 
> [Raw logs](https://github.com/OI-wiki/OI-wiki/actions/runs/10869536735/job/30160909151?pr=5855):
> 
> ```
>  --2024-09-15 08:34:46--  https://mirror.math.princeton.edu/pub/CTAN/fonts/cm-unicode.zip
> Resolving mirror.math.princeton.edu (mirror.math.princeton.edu)... 128.112.18.21
> Connecting to mirror.math.princeton.edu (mirror.math.princeton.edu)|128.112.18.21|:443... connected.
> ERROR: cannot verify mirror.math.princeton.edu's certificate, issued by 'CN=InCommon RSA Server CA 2,O=Internet2,C=US':
>   Unable to locally verify the issuer's authority.
> To connect to mirror.math.princeton.edu insecurely, use `--no-check-certificate'.
> Error: Process completed with exit code 5.
> ```

File: docs/basic/construction.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ author: leoleoasd, yzxoi
 
 ### 例题 4
 
-???+ note "BZOJ 4971「Lydsy1708 月赛」记忆中的背包"
+???+ note "[BZOJ 4971「Lydsy1708 月赛」记忆中的背包](https://vjudge.net/problem/%E9%BB%91%E6%9A%97%E7%88%86%E7%82%B8-4971)"
```
Comment (HeRaNO) [inline docs/basic/construction.md:104]:
```suggestion
???+ note "[BZOJ 4971「Lydsy1708 月赛」记忆中的背包](https://vjudge.net/problem/BZOJ-4971)"
```

**Review by HeRaNO — APPROVED** [review]

**Comment by Estrella-Explore** [comment]
@HeRaNO  tysm

**Pull request merged**